src/schema.py

```diff
@@ -115,9 +115,11 @@ def resolve_account_info(self, info, session_id=None):
       elif any(meal_swipe_name in acct['accountDisplayName'] for meal_swipe_name in SWIPE_PLANS):
         account_info['swipes'] = str(acct['balance'])
 
-    # Check if the meal plan has more than 50 swipes, this is larger than the largest plan.
-    if (int(account_info['swipes']) > 50):
-      account_info['swipes'] = 'Unlimited'
+    # Check if the balance provided by Cornell Dining is a regular digit
+    if (account_info['swipes'].isdigit()):
+      # Check if the meal plan has more than 50 swipes, this is larger than the largest plan.
+      if (int(account_info['swipes']) > 50):
```

Comment:
i know we closed this, but it's non-idiomatic to have parentheses here and in the above `if` statement!